ramalama/common.py

```diff
@@ -69,7 +69,11 @@ def run_cmd(args, cwd=None, ignore_stderr=False):
     if ignore_stderr:
         stderr=subprocess.PIPE
 
-    return subprocess.run(args, check=True, cwd=cwd, stdout=subprocess.PIPE, stderr=stderr)
+    try:
+        return subprocess.run(args, check=True, cwd=cwd, stdout=subprocess.PIPE, stderr=stderr)
+    except Exception:
+        perror(f"subprocess.run({args}, check=True, cwd={cwd}, stdout=subprocess.PIPE, stderr=stderr)")
```

Comment:
This line here is for debugging?  It is causing tests to fail. The docker error is caught above and ignored.

Reply:
I'll remove it, it was for debugging, I was hoping to keep it in, but will remove for now